packages/cli/package.json

```diff
@@ -21,13 +21,15 @@
     "node": ">=8.0.0"
   },
   "scripts": {
-    "build": "rm -rf lib && tsc -b",
+    "build": "rm -rf lib && yarn run prepack",
```

Comment:
dont run prepack on build

Reply:
I was reviving the old codebase. Removed.